cmd/rpc.go

```diff
@@ -44,13 +43,7 @@ func RPCFlags() *flag.FlagSet {
 	)
 
 	storeFlag := NodeFlags().Lookup(nodeStoreFlag)
-
-	fset.StringVar(
-		&storePath,
-		nodeStoreFlag,
-		"",
-		storeFlag.Usage,
-	)
+	fset.AddFlag(storeFlag)
```

Comment:
@vgonkivs did you change the structure of the node store flag so that it doesn't store directly to that variable?

Reply:
`node.store ` flag uses the internal field of the` *flag.Flag`. It did not use an additional var. And for the RPC cmds, yes, I removed that internal variable and now using the value directly from the flag(as you suggested):

```
if cmd.Flag(nodeStoreFlag).Changed {
		storePath = cmd.Flag(nodeStoreFlag).Value.String()	
	}
```